config/helm/chart/default/values.yaml

```diff
@@ -47,6 +50,9 @@ webhook:
   labels: []
   annotations: []
   apparmor: false
+  securityContext:
```

Comment:
I think we should set the whole security context, instead of just the seccomp profile - similar to `resources.requests/limits`

Reply:
the only problem with that is you can really easily mess it up for the csi-driver

not really for the rest

Reply:
You can already set the `resources` field to a value that will now allow the operator to work correctly. The only problem I see is that it's a bit different for the `securityContext`, since you may only want to change a single value, but would have to define all other values as well + figure out what the default values even are...